src/api/endpoints/redemption_requests.py

```diff
@@ -1,9 +1,9 @@
 """RedemptionRequest Module."""
 
-from flask import g, request
-from flask_restful import Resource
-
+from flask import request, g, current_app
+from flask_restplus import Resource
```

Comment:
It seems this was removed since swagger was not being used to document the app.

Reply:
Indeed